### PR TITLE
Made appIcon

### PR DESCRIPTION
**Pull request by Robbe95**

### CODE REVIEW

**Comment by Tanya-Amber-L** [comment]
Would it also be possible to add an overview of all available Icons in the Design System? 
Idk if it's relevant in the case of a generic-components website, but if we also want to generalize Wisemen icons used, might be good to add a list of common generic icons.

File: public/api/globalConfig.json [hunk 1]
```diff
@@ -20,14 +20,14 @@
...
...(--success))',\n        foreground: 'hsl(var(--success-foreground))',\n      },\n      popover: {\n        DEFAULT: 'hsl(var(--popover))',\n        foreground: 'hsl(var(--popover-foreground))',\n      },\n      card: {\n        DEFAULT: 'hsl(var(--card))',\n        foreground: 'hsl(var(--card-foreground))',\n      },\n\n      input: {\n        DEFAULT: 'hsl(var(--input))',\n        foreground: 'hsl(var(--input-foreground))',\n        placeholder: 'hsl(var(--input-placeholder))',\n        disabled: 'hsl(var(--input-disabled))',\n        border: 'hsl(var(--input-border))',\n        ring: 'hsl(var(--input-ring))',\n      },\n\n      neutral: {\n        900: 'hsl(var(--neutral-900))',\n        800: 'hsl(var(--neutral-800))',\n        700: 'hsl(var(--neutral-700))',\n        600: 'hsl(var(--neutral-600))',\n        500: 'hsl(var(--neutral-500))',\n        400: 'hsl(var(--neutral-400))',\n        300: 'hsl(var(--neutral-300))',\n        200: 'hsl(var(--neutral-200))',\n        100: 'hsl(var(--neutral-100))',\n      },\n    },\n    extend: {\n      transitionTimingFunction: {\n        'out-back': 'cubic-bezier(0.34, 1.56, 0.64, 1)',\n        'native': 'ease',\n      },\n    },\n  },\n}\n"
```
Comment (maltsavkiryl) [inline public/api/globalConfig.json:23]: wel funky om dat op deze manier volledig inline te doen :s

**Review by maltsavkiryl — APPROVED** [review]
globalConfig inline is wel heftig xd

File: public/api/globalConfig.json (hunk 1 above)
Comment (Robbe95) [reply public/api/globalConfig.json:23]: Das gwn generated voor de cli tool

**Comment by Robbe95** [comment]
> Would it also be possible to add an overview of all available Icons in the Design System? Idk if it's relevant in the case of a generic-components website, but if we also want to generalize Wisemen icons used, might be good to add a list of common generic icons.

We discussed this with the designers and gave them a list, so every design after the rock of theirs is done, should have a list of general icons we use in the our component library.

**Pull request merged**